src/tcpip/tcpip.c

```diff
@@ -598,10 +600,12 @@ static void read_conn(struct mg_connection *c, struct pkt *pkt) {
     MG_DEBUG(("%lu SEQ %x -> %x", c->id, mg_htonl(pkt->tcp->seq), s->ack));
     // Advance ACK counter
     s->ack = (uint32_t) (mg_htonl(pkt->tcp->seq) + pkt->pay.len);
-#if 0
+#ifdef MIP_TEST
```

Comment:
?
This causes MIP to send an ACK on SEQ reception, instead of waiting and possibly piggybacking it on a likely response segment. Makes it very responsive at the cost of higher traffic.

Reply:
What I don't like with this is that we might screw the timer and piggyback and won't notice... we should test the code section that will be used in real life. Perhaps with another test like mip_tap_test, please make sure we do and leave a comment so I don't panic again some months from now...